ddtrace/contrib/django/conf.py

```diff
@@ -25,6 +25,8 @@
     'AGENT_HOSTNAME': 'localhost',
     'AGENT_PORT': 8126,
     'AUTO_INSTRUMENT': True,
+    'DATABASE_AUTO_INSTRUMENT': True,
```

Comment:
if we're moving to a flexible instrumentation config, I'd say to add `AUTO_INSTRUMENT_TEMPLATE`, so that every single instrumented part of Django can be disabled. What do you think?